docs/html/reference/pip_uninstall.rst

```diff
@@ -30,11 +38,26 @@ Examples
 
 #. Uninstall a package.
 
-  ::
+  .. tabs::
+
+      .. group-tab:: Unix/macOS
```

Comment:
I've just went through the files and it seems that this (till the end of file) is the only one you missed.  Thank you for putting up with my OCD :sweat_smile:  Personally I can't wait to see this merged!

Reply:
No issues at all. Just submitted this change.